stories/src/examples/MarkerFilters.tsx

```diff
@@ -22,41 +17,84 @@ const Panel = styled.div`
 
 const setCount = 6
 
+const MarkerStyles = createGlobalStyle`
+  .custom-marker {
+    background-color: ${themeColor('primary')};
+    border-radius: 50%;
+    color: #fff;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    cursor: zoom-in;
+    text-transform: uppercase;
+  }
+`
+
+function createDatasetMarkers(datasetId: string, coordinates: LatLngTuple[]) {
+  return coordinates.map(([lat, lng]) => {
+    const html = `<span aria-label="Marker uit dataset ${datasetId} op location lat: ${lat} lng: ${lng}">${datasetId}</span>`
+    const markerObject = L.marker(new L.LatLng(lat, lng), {
+      icon: L.divIcon({
+        html,
+        className: `custom-marker custom-marker--${datasetId}`,
+        iconSize: L.point(20, 20),
+        iconAnchor: L.point(10, 10),
+      }),
+    })
+    return markerObject
+  })
+}
+
+interface MarkerFiltersProps {
+  markersSource: L.LatLngTuple[]
+  clusterComponent: (markers: any) => ReactNode
```

Comment:
can be of type `L.LatLngTuple[]` I guess, or maybe even `L.LatLng[]`

Reply:
It's actually `L.Marker[]`